scripts/o3de/tests/test_repo.py

```diff
@@ -65,27 +65,21 @@
 }
 '''
 
-# This holds the repo 2.0.0 schema, which houses all relevant objects in file
+# This holds the repo 1.0.0 schema, which houses all relevant objects in file
 # A possible future field for the version data is download_prebuilt_uri, 
 # which indicates where to download dedicated binaries of a version of the O3DE object. 
 TEST_O3DE_REPO_JSON_VERSION_2_FILENAME = '3b14717bafd5a3bd768d3d0791a44998c3bd0fb2bfa1a7e2ee8bb1a39b04d631.json'
 TEST_O3DE_REPO_JSON_VERSION_2_PAYLOAD = '''
 {
     "repo_name": "testgem3",
-
     "origin": "Studios",
-
-    "$schemaVersion":"2.0.0",
-    
+    "$schemaVersion":"1.0.0",
     "repo_uri": "https://downloads.testgem3.com/o3de-repo",
-
     "summary": "Studios Repository for the testgem3 Gem.",
-
     "additional_info": "",
-
     "last_updated": "2023-01-19",
-
-    "gems": [{
+    "gems":[],
```

Comment:
I'm not sure I understand your question, but maybe this will help, the fields in "gems", "projects" and "templates" are URLs, the "gems_data" fields contains JSON data.